iree/turbine/kernel/wave/codegen/handlers.py

```diff
@@ -465,6 +466,18 @@ def handle_and(lhs: Value, rhs: Value) -> OpResult:
     return result
 
 
+@handle_binary_op(operator.or_)
+def handle_and(lhs: Value, rhs: Value) -> OpResult:
+    element_type = get_type_or_element_type(lhs.type)
+    if _is_integer_like_type(element_type) and (
```

Comment:
Probably better to move this code int function as it used in multiple places now.

Reply:
Done. Refactored rest of the code too.